python/tvm/tir/op.py

```diff
@@ -965,6 +965,34 @@ def popcount(x):
     """
     return call_intrin(x.dtype, "tir.popcount", x)
 
+def qmuls(x, y, q, s):
+    """Execute a multiplication between two Q-numbers x and y
+    followed by a right shift s. The mathematical expression is:
+
+       out = round(x*y*2^-s)
```

Comment:
Maybe we should add a line to explain why there is a multiplication factor of 2 (perhaps rounding)

Reply:
I am not sure I follow. Do you mean the multiplication by `2^-s`?

Reply:
Oh I completely misread. This is good.